### PR TITLE
Medical Item Resprites

### PR DESCRIPTION
**Pull request by Coconutwarrior97**
:cl: 
tweak: If you rename medical sprays to stypic powder, silver sulfadine, or synthflesh it updates the icon to the appropriate spray type. Also when making patches in the chem master if they are named "brute" or "burn" they have their icon changed to the brute/burn patches.
/:cl:

Spray renames for medical doctors if they feel like making some sprays are fun.

### CODE REVIEW

File: code/modules/reagents/reagent_containers/medspray.dm [hunk 1]
```diff
@@ -89,3 +89,14 @@
 	name = "sterilizer spray"
 	desc = "Spray bottle loaded with non-toxic sterilizer. Useful in preparation for surgery."
 	list_reagents = list("sterilizine" = 60)
+/obj/item/reagent_containers/medspray/changed_name(newname)
+	if (lowertext(newname) == ("stypic powder"))
+		icon_state = "brutespray"
+		update_icon()
+	if (lowertext(newname) == ("silver sulfadine"))			// Renamed sprays have their icon changed to fit their new name when appropriate.
```
Comment (subject217) [inline code/modules/reagents/reagent_containers/medspray.dm]: Silver Sulfadiazine

**Review by lilacrhododendron — CHANGES_REQUESTED** [review]

File: code/modules/reagents/reagent_containers/medspray.dm (hunk 1 above)
Comment (Coconutwarrior97) [reply code/modules/reagents/reagent_containers/medspray.dm]: Its fixed.

**Comment by Mark9013100** [comment]
Could this be applied to the medical patches as well?

File: code/modules/reagents/reagent_containers/medspray.dm [hunk 2]
```diff
@@ -89,3 +89,14 @@
 	name = "sterilizer spray"
 	desc = "Spray bottle loaded with non-toxic sterilizer. Useful in preparation for surgery."
 	list_reagents = list("sterilizine" = 60)
+/obj/item/reagent_containers/medspray/changed_name(newname)
+	if (lowertext(newname) == ("stypic powder"))
```
Comment (Time-Green) [inline code/modules/reagents/reagent_containers/medspray.dm]: styptic

Comment (Coconutwarrior97) [reply code/modules/reagents/reagent_containers/medspray.dm]: Fixed.

**Comment by Coconutwarrior97** [comment]
> 
> 
> Could this be applied to the medical patches as well?

I'll add this tomorrow.

**Comment by Tlaltecuhtli** [comment]
can this be applied to pills?

**Comment by FantasticFwoosh** [comment]
You can already rename spray cans, this is RESPRITE (like cardboard juiceboxes becoming milk boxes) so adjust the title please for your broken english.

**Comment by WarRage333** [comment]
Now custom spray can that have the fire icon on them can actually apply fire to people 

I love it

File: code/modules/reagents/reagent_containers/medspray.dm [hunk 3]
```diff
@@ -91,3 +91,14 @@
 	name = "sterilizer spray"
 	desc = "Spray bottle loaded with non-toxic sterilizer. Useful in preparation for surgery."
 	list_reagents = list("sterilizine" = 60)
+/obj/item/reagent_containers/medspray/changed_name(newname)
+	if (lowertext(newname) == ("styptic powder"))
```
Comment (optimumtact) [inline code/modules/reagents/reagent_containers/medspray.dm]: don't constantly call lowertext

just do it once at the start of the proc
var/compare = lowertext(newname)
if(name == "styptic powder") 
        icon state

also don't call update icon every single time, just put it right at the bottom of the if chain once

**Comment by Coconutwarrior97** [comment]
Im closing this PR because it turns out this branch is somewhat out of date, caused some annoying issues so its easier to just open a new PR.

**Pull request closed without merging**